phases/19-capstone-projects/15-constitutional-safety-harness/code/main.py

```
@@ -0,0 +1,246 @@
+                text = text.replace(tok, decoded)
+        except Exception:
+            pass
+    return text
+
+
+# ---------------------------------------------------------------------------
+# classifier gate  --  Llama Guard 4 / X-Guard / ShieldGemma stand-ins
+# ---------------------------------------------------------------------------
+
+BAD_EN = {"bomb", "malware", "suicide", "exploit", "meth"}
+BAD_MULTILINGUAL = {"bombe", "bombe", "malware", "bomba", "nitroglycerin"}
```

Ruff (0.15.11)

[warning] Sets should not contain duplicate item `"bombe"`

Remove duplicate item

(B033)